sigopt/magics.py

```diff
@@ -26,10 +26,10 @@ def get_ns():
   stack_depth = 1
   while True:
     frame = sys._getframe(stack_depth)
-    locals = frame.f_locals
+    f_locals = frame.f_locals
     try:
       if isinstance(locals['exit'], IPython.core.autocall.ExitAutocall):
```

Comment:
need to change `locals` to `f_locals` on this line

Reply:
i looked at that line like 3 times and kept missing it. thanks.